### DISCUSSION TITLE
Illegal instruction(code dumped)

### DISCUSSION THREAD

**Discussion by lorch** [openssl/openssl#21760] [Q&A]
Hi, 

OS : Linux jh-web-services 5.10.134-15.al8.x86_64 #1 SMP Thu Jul 20 00:44:04 CST 2023 x86_64 x86_64 x86_64 GNU/Linux

Rust openssl : openssl 0.10.55

Run bin file  show Illegal instruction(code dumped)

how to fix it ?

**Comment by nhorman** [comment]
What makes you think the illegal instruction was in the openssl code?  Do you have a log message or stack trace that suggests such?
